ui/src/app/app.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent {
+  title = 'dragon';
```

Comment:
Same comment.
Is this supposed to be hardcoded or replaced by the environment variable?

Reply:
These are the angular application title. It not meant to be replaced by environment variables.